### ISSUE TITLE
v21.0.0.11-beta failed my sample project

### ISSUE THREAD

**Issue by hantsy** [OpenLiberty/open-liberty#18822]
I created [hantsy/jakartaee9-starter-boilerplate](https://github.com/hantsy/jakartaee9-starter-boilerplate) to demo the latest Jakarta EE 9, in former versions, all tests are working well on OpenLiberty beta(EE9). 

**Diagnostic information:**  
 - OpenLiberty Version: v21.0.0.11-beta
 - CI result :  https://github.com/hantsy/jakartaee9-starter-boilerplate/pull/177#partial-pull-merging

**Comment by tevans78** [comment]
@hantsy Please can you be more specific? All I can see is
```
 Error:  Failures: 
Error:    GreetingResourceTest.should_create_greeting:74 
expected: 200
 but was: 404
Error:  Errors: 
Error:    GreetingServiceTest.should_create_greeting:40 NullPointer
[INFO] 
Error:  Tests run: 2, Failures: 1, Errors: 1, Skipped: 0
```
Please can you post the server logs?

**Comment by jhanders34** [comment]
The problem here is that you specified jakartaee-9.0 in your server.xml.  In 21.0.0.11-beta, jakartaee-9.0 was replaced with jakartaee-9.1.  This was announced previously in the beta blogs that this was going to happen.  The full beta image (as compared to the jakartaee9 beta image) had both jakartaee-9.0 and 9.1 until this new beta version.  The beta blog has not been published yet for 21.0.0.11-beta, but it will include this information about the jakartaee-9.0 going away and replaced by jakartaee-9.1.  There should be a nice error in the messages.log indicating to change it from 9.0 to 9.1.  Something like:

[10/1/21 7:29:29:498 CDT] 00000020 com.ibm.ws.kernel.feature.internal.FeatureManager            E CWWKF0001E: A feature definition could not be found for jakartaee-9.0
[10/1/21 7:29:29:499 CDT] 00000020 com.ibm.ws.kernel.feature.internal.FeatureManager            E CWWKF0045E: An existing feature definition, jakartaee-9.1, is a possible match for the jakartaee-9.0 feature that was not found.

**Comment by hantsy** [comment]
@jhanders34  This fixed my issues. Thanks.
